<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainNunberChangeWidget</class>
 <widget class="QWidget" name="TrainNunberChangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置车次</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	padding-left: 8px;
	border-top:2px inset rgb(180, 180, 180);
	border-left:2px inset rgb(180, 180, 180);
	border-bottom:2px inset rgb(240, 240, 240);
	border-right:2px inset rgb(240, 240, 240);
}

QLineEdit:!enabled{
	background-color: rgb(240, 240, 240);
	padding-left: 8px;
	border-top:2px inset rgb(180, 180, 180);
	border-left:2px inset rgb(180, 180, 180);
	border-bottom:2px inset rgb(240, 240, 240);
	border-right:2px inset rgb(240, 240, 240);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>请选择车站</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>原列车信息</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>到达车次:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>出发车次:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>新列车信息</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>到达车次:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>出发车次:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>36</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>56</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>保留车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>56</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>终到车</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>36</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>确定(O)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>取消(C)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
